### PR TITLE
Remove Coffee artifacts, updating to pulsar

### PR DESCRIPTION
**Pull request by confused-Techie**
This PR essentially rebrands this package, and is a needed step if we want to start maintaining our own version on the package registry.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
     "type": "git",
-    "url": "https://github.com/atom/language-pegjs.git"
+    "url": "https://github.com/pulsar-edit/language-pegjs.git"
   },
   "bugs": {
-    "url": "https://github.com/atom/language-pegjs/issues"
-  },
-  "devDependencies": {
-    "coffeelint": "^1.10.1"
+    "url": "https://github.com/pulsar-edit/language-pegjs/issues"
```
Comment (sertonix) [inline package.json]: To make things more in line with other `package.json`s I would suggest removing the `"homepage"` and `"bugs"` values and change the `"repository"` value to `https://github.com/pulsar-edit/language-pegjs`.

Edit: Searching some more I saw that there is no `license` value. May be nice to add that.

Comment (confused-Techie) [reply package.json]: All fantastic ideas, lemme address those. Thanks a ton for the review @Sertonix! Always appreciate your input, and love to see you pop up in my notifications again

Comment (sertonix) [reply package.json]: I hope to get a bit more active soon.

**Comment by confused-Techie** [comment]
Addressed your concerns above @Sertonix

**Review by sertonix — APPROVED** [review]

**Pull request merged**